widgets/google-map/google-map.php

```diff
@@ -492,13 +525,23 @@ function get_template_variables( $instance, $args ) {
 					}
 				}
 			}
+      
 			$location = $this->get_location_string( $instance['map_center'] );
-			
+      
+      // Work out what we need to set gestureHandling to
+			if ( ! $settings['draggable'] ) {
+				$gestureHandling = 'none';
+			} elseif ( ! $settings['scroll_zoom'] ) {
+				$gestureHandling = 'cooperative';
+			} else {
+				$gestureHandling = 'greedy';
+			}
```

Comment:
Looking at this again with fresh eyes it occurred to me that with `draggable` and `scrollwheel` being deprecated we should migrate those to a single `gestureHandling` radio field in the form to stay consistent with the Maps API. This code is effectively migrating those settings anyway.

Reply:
That's an interesting idea. How do these options look:

- Enabled (greedy)
- Dynamically change based on device (auto)
- ctrl key or ⌘ click to scroll (cooperative)
- Disabled (none)

The problem with this change is that it makes it quite tricky to add an inline description without it being really large. Hm. Actually, we could just link to the documentation. Something as "For information on what these settings do, click here.". It's not ideal but it doesn't matter how long the documentation is.

Reply:
Yeah, I think we can label the options exactly as they are in the Google Maps docs [here](https://developers.google.com/maps/documentation/javascript/interaction#gestureHandling) e.g: 

- Greedy
- Cooperative
- None
- Auto

and then that description looks good _"For information on what these settings do, click here."_ to link to the Google Maps docs, as an explanation to users.